### PR TITLE
Optimize performance on creating schemas when exporting to Big Query

### PR DESCRIPTION
**Pull request by ycd**
Refactored the code, replaced iteration with a list comprehension.

### CODE REVIEW

**Comment by ycd** [comment]
@googlebot I signed it!

File: slo_generator/exporters/bigquery.py [hunk 1]
```diff
@@ -82,14 +82,13 @@ def create_table(self, project_id, dataset_id, table_id, schema=None):
...
-        pyschema = []
         if schema is not None:
             schema = TABLE_SCHEMA
-        for row in schema:
-            field = bigquery.SchemaField(row['name'],
+        pyschema = [bigquery.SchemaField(row['name'],
                                          row['type'],
                                          mode=row['mode'])
-            pyschema.append(field)
+                                         for row in schema]
```
Comment (ocervell) [inline slo_generator/exporters/bigquery.py:90]: this is over-indented according to pylint, can you please correct it ?

**Review by ocervell — CHANGES_REQUESTED** [review]
Thanks ! Just a small knit to fix visual indent and we're good to go. You can run lint tests using `make install_test flake8 pylint`.

File: slo_generator/exporters/bigquery.py (hunk 1 above)
Comment (ycd) [reply slo_generator/exporters/bigquery.py]: @ocervell done!

**Pull request merged**